spec/service_consumers/pact_helper.rb

```diff
@@ -44,4 +46,43 @@ def url_encode(str)
       create(:place, service_slug: service.slug, latitude: 50.742754933617285, longitude: -1.9552618901330387)
     end
   end
+
+  provider_state "a place exists with a postcode and areas" do
+    set_up do
+      service = create(:service, slug: "local-authority")
+      create(:place, service_slug: service.slug, postcode: "WC2B 6SE")
+
+      areas = [
+        { "name" => "Westminster City Council", "type" => "LBO" },
+        { "name" => "London", "type" => "EUR" },
+      ]
+
+      response = {
+        "wgs84_lat" => 51.516,
+        "wgs84_lon" => -0.121,
+        "postcode" => "WC2B 6SE",
+      }
+
+      area_response = Hash[areas.map.with_index do |area, i|
+        [i,
+         {
+           "codes" => {
+             "ons" => area["ons"],
+             "gss" => area["gss"],
+             "govuk_slug" => area["govuk_slug"],
+           },
+           "name" => area["name"],
+           "type" => area["type"],
+           "country_name" => area["country_name"],
+         }]
+      end]
+
+      postcode = "WC2B 6SE"
+
+      stub_request(:get, "#{MAPIT_ENDPOINT}/postcode/#{postcode.tr(' ', '+')}.json")
```

Comment:
`sub` is more idiomatic:

```suggestion
      stub_request(:get, "#{MAPIT_ENDPOINT}/postcode/#{postcode.sub(' ', '+')}.json")
```

Reply:
👍